### ISSUE TITLE
DL3016 does not support -g switch but only --global

### ISSUE THREAD

**Issue by ferrarimarco** [hadolint/hadolint#151]
- [x] This is a bug report
- [ ] This is a feature request
- [x] I searched existing issues before opening this one

### Expected behavior
`RUN npm install -g gulp@3.9.1` should be fine, but it's not. According to the [npm docs](https://docs.npmjs.com/cli/npm), either `-g` or `--global` can be used to install a package globally.

### Actual behavior

`Dockerfile:58 DL3016 Pin versions in npm. Instead of `npm install <package>` use `npm install <package>@<version>``

### Steps to reproduce the behavior

`docker run -v "$(pwd)":/mnt --rm -w="/mnt" hadolint/hadolint:v1.2.6 hadolint Dockerfile`

**Output of `docker run --rm hadolint/hadolint hadolint --version`:** `Haskell Dockerfile Linter v1.2.6-0-ge03cbb0`

```bash
Dockerfile:58 DL3016 Pin versions in npm. Instead of `npm install <package>` use `npm install <package>@<version>`
```

**Dockerfile** (if relevant)

```Dockerfile
FROM ubuntu:16.04
ENV NVM_DIR /usr/local/nvm
ENV NVM_VERSION 0.33.8
RUN curl -o- "https://raw.githubusercontent.com/creationix/nvm/v$NVM_VERSION/install.sh" | bash

RUN \
  . "$NVM_DIR"/nvm.sh \
  && nvm install v0.10.25 \
  && npm install -g gulp@3.9.1
```
